### ISSUE TITLE
ETag/If-Match/If-None-Match problems

### ISSUE THREAD

**Issue by fugu13** [adlnet/xAPI-Spec#356]
I've noticed something in our ETag-related language: we say we're relying on the appropriate RFC (2616, http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html ), but then we say the appropriate response when If-Match or If-None-Match triggers is 409 in several (though not all) places, however, that's not a conformant response by the RFC: the only allowed conformant response to those particular conditions is 412.

Given that this is well-traveled RFC territory with extensive library support built around 412 status codes, once we've had a chance to update (we're actually a bit inconsistent at the moment depending on which reference we were looking at when implementing) Wax will be responding with 412 in such situations.

In other words, Saltbox is treating this as "it says strongly that the RFC controls, so the mentions of 409 elsewhere are typos" right now; that seems like the right approach to me.
